tests/PHPUnit/System/expected/test_AutoSuggestAPITest_deviceBrand__API.getSuggestedValuesForSegment.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<result>
+	<row>Unknown</row>
```

Comment:
@sgiehl would it be possible to have this test generate real values so it shows that the auto suggest for this new segment works as expected?